app/Libraries/TeamSpeak.php

```diff
@@ -359,6 +360,11 @@ public static function checkClientIdleTime(TeamSpeak3_Node_Client $client, Accou
             $maxIdleTime = 60;
         }
 
+        if($server->channelGetById($client->cid)->getId() == 1){
```

Comment:
The channel ID should be the default channel ID, rather than having `1` hardcoded. The default channel can be obtained using `$client->getParent()->channelList(['channel_flag_default' => 1])`